README.md

```diff
@@ -36,7 +36,7 @@ The executable jar can be executed with a command like `java -jar dsbulk-distrib
 
 ## Documentation
 
-The most up-to-date documentation is available [online][onlineDocs]. 
+The most up-to-date documentation is available [online](https://docs.datastax.com/en/dsbulk/docs/1.9.0/dsbulkAbout.html). 
```

Comment:
Since you are inlining the link, please remove the "footpage link" at the bottom of the page:

```
[onlineDocs]:https://docs.datastax.com/en/dsbulk/doc/
```

Reply:
Also, couldn't we avoid hard-coding 1.9 in the URL? E.g. by using a "latest" redirect?

Reply:
Regarding the /latest/ instead of /1.9.0/, that proposed scheme was objected to by a prior generation of doc leads. Something about customers assuming their version was the "latest" and complained to Support that they couldn't get a doc'd feature (introduced in a more recent release) to work. For the moment I'll need to hardcode the /1.9.0/ but you raise a fair point.

Reply:
Yes will remove the [onlineDocs] reference from the footer.  Thanks for spotting it.

Reply:
I understand the customers objections. But in the present case, we are considering DSBulk's own in-tree README file. Moreover, the link is being described as "the most up-to-date documentation". If we hard-code 1.9.0 here, the README file and this description will become outdated as soon as we publish a 1.10 release. Are we sure this is what we want?